appstore/appstore/settings/base.py

```diff
@@ -349,6 +356,11 @@
         "127.0.0.1",
     ]
 
+    CSRF_TRUSTED_ORIGINS += [
+        "https://localhost",
```

Comment:
We sure SSL will be enabled in this particular setup? "https"?

Reply:
Yeah, this should be http/both http and https

Reply:
Fixed.